src/components/Footer/footer.js

```diff
@@ -34,6 +42,11 @@ const Footer = () => (
                             Email us
                         </a>
                     </li>
+                    <li>
+                        <a href="/code-of-conduct" title="Code of conduct">
+                            Code of conduct
```

Comment:
If the title is the same as the text, is it needed?

Reply:
No it's not, but all the other links had it, so I followed the convention. Would prefer to remove the title text from all the footer links actually.

Reply:
If you're happy to do that, then go for it. Will leave it up to you :)